CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### Changelog
 
+## VERSION 6.12.0 (Aug 1, 2022)
```

Comment:
@ashinagawa Change version to 6.12.**1** and date to Nov 7, 2022

Reply:
Thanks. I updated it.